### PR TITLE
added make file to facilitate some complex docker commands.

### PR DESCRIPTION
**Pull request by knollfear**
I think long (and even short) term this will be easier to use and maintain.  People are under no obligation to use this.  They can keep doing things exactly the way they currently are.

### CODE REVIEW

**Comment by MothOnMars** [comment]
Thanks, @knollfear . Can you add some info to the bottom of the "Elasticsearch" section of the README: https://github.com/GSA/search-gov#elasticsearch ? You can add it after the section on running ES 1.7, which is about to be deprecated.

**Comment by knollfear** [comment]
> Thanks, @knollfear . Can you add some info to the bottom of the "Elasticsearch" section of the README: https://github.com/GSA/search-gov#elasticsearch ? You can add it after the section on running ES 1.7, which is about to be deprecated.

I added in some words about using the makefile to the end of that section.

File: README.markdown [hunk 1]
```diff
@@ -187,6 +187,17 @@ Now restart it:
 
 If you aren't using Homebrew to install and configure Elasticsearch, follow the [instructions](http://www.elasticsearch.org/download/) to download and run it.
 
+In addition, Elastic search 5.6 is also required.  Fortunately, this version is dockerized and can be run locally entirely within a docker container.
```
Comment (MothOnMars) [inline README.markdown]: "Elasticsearch" is one word

File: README.markdown [hunk 2]
```diff
@@ -187,6 +187,17 @@ Now restart it:
...
 If you aren't using Homebrew to install and configure Elasticsearch, follow the [instructions](http://www.elasticsearch.org/download/) to download and run it.
 
+In addition, Elastic search 5.6 is also required.  Fortunately, this version is dockerized and can be run locally entirely within a docker container.  
+To facilitate use of this container a makefile has been provided with the following 3 commands:
+    
+    make pull-elastic - This pulls the desired container from dockerhub
+    
+    make run-elastic - Runs the docker container with the correct ports
+    
+    make test-elastic - Tests that the elastic search container is currently running
```
Comment (MothOnMars) [inline README.markdown]: Command is `check-elastic`

**Pull request merged**